com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/TargetData/SystemData.cs

```diff
@@ -127,16 +127,19 @@ public ShaderGraphVersion version
         }
 
         [SerializeField]
-        bool m_FirstTimeMigrationExecuted = false;
```

Comment:
This is no longer required as explained in the doc. See usage of OnBeforeDeserialize()